Installing-Oppia-(Linux;-Python-3).md

```diff
@@ -143,26 +143,32 @@ For your virtual environment, we recommend you use [pyenv](https://github.com/py
    Installed Python-3.7.10 to /home/user/.pyenv/versions/3.7.10
    ```
 
-4. Create a virtual environment for oppia:
+5. Install direnv
 
-   ```console
-   $ pyenv virtualenv 3.7.10 oppia
-   ...
-   $ pyenv versions
-   ...
-   oppia
-   ...
+   ```sh
+   $ sudo apt install direnv
+   ```
+   
+6. Setup direnv into your shell. Add following lines to the end of `.bashrc` (see [here](https://askubuntu.com/a/127059) for where to find this file):
+   ```bash
+   eval "$(direnv hook bash)"
    ```
 
-   In the cloned `oppia` folder, run
+7. Create a virtual environment for oppia. Add file named `.envrc` into the parent folder of the oppia repository with this content:
 
    ```console
-   pyenv local oppia
+   use python 3.7.10
```

Comment:
How does this specify that the virtual environment (and not just the global python 3.7.10 installation) should be used?

Reply:
Ah, I forgot one step before this.